src/components/Table/Table.md

```diff
@@ -0,0 +1,21 @@
+# Table
+Presents tabular data with multiple rows and columns.
```

Comment:
It may be worth mentioning that, although this component is responsive to an extent, it is not *completely* responsive (e.g. collapsing columns into single items for maximum readability).